scripts/build_etcd_test_analyzer.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
```

Comment:
```suggestion
```

Not needed to comment it, we are adding script bash everywhere https://github.com/etcd-io/etcd/pull/15519.